src/providers/hive/dhive.js

```diff
@@ -740,11 +740,15 @@ const getBlockNum = async (trx_id) => {
 export const vote = async (account, pin, author, permlink, weight) => {
   try {
     const resp = await _vote(account, pin, author, permlink, weight);
+    console.log('Returning vote response', resp);
+    if (!resp || !resp.id) {
+      return Promise.reject(new Error('Invalid response received from hive'));
```

Comment:
Phrase could be translated and also `Invalid response received from server` maybe be better.

Reply:
done